### DISCUSSION TITLE
Julia plotting (3D...)

### DISCUSSION THREAD

**Discussion by jbcaillau** [control-toolbox/control-toolbox.github.io#27] [Ideas]
Check
- [PyplotJS.jl](https://discourse.julialang.org/t/rotatable-3d-plot-in-jupyter/70535/2)
- [Mackie.jl](https://docs.makie.org/stable/)

![5A4E61B6-B53C-464B-82F7-2FC53A331EEE](https://user-images.githubusercontent.com/62183989/228379291-e8eb3f3a-25b1-4c23-8557-de279bb150ee.jpeg)

https://user-images.githubusercontent.com/62183989/228379319-509c0939-9264-4aac-943b-9808126b4149.mp4

**Comment by ocots** [comment]
I tried twice but I never managed to install `Makie.jl`!

**Reply by jbcaillau** [reply]
obi-wan has taught you well but you are not a jedi yet
